cygnus-ngsi/test/performance/loadTest_cygnus_mock_v1.0.jmx

```diff
@@ -1,25 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!--
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1230/commits/8fc4664b2dc6f2f5713e0e518508dc264b2fae4f